kiosk/src/Components/MainMenu.tsx

```diff
@@ -21,6 +22,7 @@ const MainMenu: React.FC<IProps> = ({ kiosk }) => {
             setButtonState(false);
         }
         if (buttonSelected && kiosk.gamepadManager.isAButtonPressed()) {
+            tickEvent("kiosk.addGamePageLoaded");
```

Comment:
I would add the information on which game is loaded? That would be useful to see the popularity of the games. You can use parameters to the tickEvent function to send customDimensions.

Reply:
Yeah, I do that here: https://github.com/microsoft/pxt-arcade/pull/5507/files#diff-0cd255ed560e0ea2631bc7d448d834a3ae74873998e86ade52d835d937fd931bR65

Reply:
I can also add other game details if that is valuable alongside game id.